src/Resources/views/form_layout.html.twig

```diff
@@ -44,7 +44,7 @@
     {%- if errors|length > 0 -%}
         <ul>
             {%- for error in errors -%}
-                <li class="mb-2 mt-2 whatwedo-utility-paragraph--error py-3 relative">{{ error.message|trans }}</li>
+                <li class="whatwedo-utility-paragraph--error relative">{{ error.message|trans }}</li>
```

Comment:
Nit: I see different formatting used in different MRs for Twig filters. Some have spaces between the pipe, others don't. This is out of scope for this MR, but it seems the formatting rules are off here.

Reply:
I created a issue in the Meta repo to keep track of that. Will make sense during the refactoring of the components anyways